### ISSUE TITLE
White space when setPotrait

### ISSUE THREAD

**Issue by hifiaz** [RafaelBarbosatec/bonfire#9]
hello, great work i love this

but i have some problem, don't know it's bug or something wrong, so when set layout to potrait there have space with white color, in main i have set black, but just comeback to horizontal the no more white spare

<img width="715" alt="Screen Shot 2020-04-08 at 18 26 31" src="https://user-images.githubusercontent.com/20653803/78779389-14a48300-79c7-11ea-870b-2e5f2927f1d8.png">
<img width="322" alt="Screen Shot 2020-04-08 at 18 27 06" src="https://user-images.githubusercontent.com/20653803/78779397-179f7380-79c7-11ea-9f78-cd1df379bda3.png">

**Comment by RafaelBarbosatec** [comment]
Thanks!
I will test this. But that looks like the buttons on the android system.
Or the game's widget is not covering the entire screen.

**Comment by RafaelBarbosatec** [comment]
Sorry, you can't reproduce the problem. Try to update the package, if it doesn't work send the code here to look better.

**Comment by hifiaz** [comment]
sorry still found that after update,

 thanks for your response

```
import 'package:bonfire/bonfire.dart';
import 'package:flutter/material.dart';
import 'package:perang_pesawat/map.dart';
import 'package:perang_pesawat/ship.dart';

class Game extends StatefulWidget {
  @override
  _GameState createState() => _GameState();
}

class _GameState extends State<Game> {
  bool showGameOver = false;

  @override
  Widget build(BuildContext context) {
    return BonfireWidget(
      // background: BackgroundColorGame(Colors.white),
      joystick: Joystick(
        pathSpriteBackgroundDirectional: 'joystick_background.png',
        pathSpriteKnobDirectional: 'joystick_knob.png',
        sizeDirectional: 100,
        actions: [
          JoystickAction(
            actionId: 1,
            pathSprite: 'joystick_atack_range.png',
            pathSpritePressed: 'joystick_atack_range_selected.png',
            size: 50,
            margin: EdgeInsets.only(bottom: 50)
          )
        ],
      ),
      map: PesawatMap.map(),
      player: Ship(
        initPosition: Position(5 * 32.0, 10 * 32.0),
      ),
    );
  }
}
```

[full source code](https://github.com/hifiaz/pesawat-tempur)

**Comment by RafaelBarbosatec** [comment]
![photo4954507898973694066](https://user-images.githubusercontent.com/10039275/79406145-a6366680-7f6c-11ea-8c2d-f7a0e3efba95.jpg)

I downloaded your project to test. cool.

I exec on my device. and apparently it’s normal.
I had to comment this line:

```dart
visualDensity: VisualDensity.adaptivePlatformDensity
```

but I don't know if that influences in this problem.

**Comment by RafaelBarbosatec** [comment]
I was able to identify a problem. I will upload a new version with this improvement

**Comment by RafaelBarbosatec** [comment]
Done! improvement available in version 0.1.9.
